browser-sign-in/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "@okta/okta-react-native": "2.1.1",
+    "@okta/okta-react-native": "2.3.0",
     "events": "^3.2.0",
     "react": "17.0.1",
     "react-native": "^0.64.1",
```

Comment:
Is it possible to update these dependencies to their latest releases too?

Reply:
Well, it looks like we do not have full 0.65+ support on android yet, we have to update okta-react-native first